examples/bigcommerce/lib/bigcommerce/defaults.ts

```diff
@@ -0,0 +1,48 @@
+import { ProductFragment } from "./types";
```

Comment:
@migueloller this was my best guess at extending the pattern established in the other files